package.json

```diff
@@ -18,7 +18,9 @@
     "coveralls": "^3.0.0",
     "cz-conventional-changelog": "^2.1.0",
     "husky": "^0.14.3",
-    "lerna": "^2.9.1"
+    "lerna": "^2.9.1",
+    "request": "^2.87.0",
```

Comment:
Why do we install these modules at the monorepo level instead of at the package level?

Reply:
oops, must've forgot to `cd` into the package before installing. fixing.